cypress/integration/promiseFunction.spec.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+Cypress.Commands.add('hasPromise', (promise, element, id) => {
```

Comment:
Pormise is a javascript object with three states: pending, resolved, rejected. 

How can page 'havePromise'? Let's chat offline about a better name.

Reply:
renamed